views/cypress/tests/import-export.spec.js

```diff
@@ -63,19 +65,31 @@ describe('Import/export items', () => {
                 selectors.deleteClassUrl,
                 true
             );
-        };
-
-        it('can import item', function () {
-            importItemTest('e2e_item.zip');
         });
 
-        it('can import item with shared stimulus', function () {
-            importItemTest('e2e_item_shared_stimulus.zip');
+        [{
```

Comment:
Where is the CSV import?

Reply:
There is no csv import on the UI

Reply:
It is in the description of the main ticket. Maybe it was about metadata import only, but we only export it (unless we are talking about tabular import - which is CSV).